diracx-db/src/diracx/db/sql/pilot_agents/db.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from datetime import datetime, timezone
+
+from sqlalchemy import insert
+
+from ..utils import BaseSQLDB
+from .schema import PilotAgents, PilotAgentsDBBase
+
+
+class PilotAgentsDB(BaseSQLDB):
+    metadata = PilotAgentsDBBase.metadata
+
+    async def addPilotReferences(
+        self,
+        pilotRef: list[str],
```

Comment:
We want to use snake case in diracx: 
```suggestion
        pilot_ref: list[str],
```